workers/unity/Packages/io.improbable.gdk.core/Subscriptions/CommandSenderReceiverSubscriptionManagerBase.cs

```diff
@@ -196,8 +199,8 @@ public override Subscription<T> Subscribe(EntityId entityId)
             }
 
             if (WorkerSystem.TryGetEntity(entityId, out var entity)
-                && componentUpdateSystem.HasComponent(componentId, entityId)
-                && componentUpdateSystem.GetAuthority(entityId, componentId) != Authority.NotAuthoritative)
+                && entityManager.HasComponent(entity, componentType)
```

Comment:
Is this actually required now? The presence of the Auth component is sufficient right?